app/jobs/twilio_job.rb

```diff
@@ -1,6 +1,10 @@
 class TwilioJob  < ActiveJob::Base
   queue_as :default
 
+  rescue_from(Twilio::REST::RequestError) do
+    retry_job wait: 30.seconds
+  end
```

Comment:
@swatijadhav If this retries every 30 seconds, is there anything that will make it stop after X times?